client/web/src/components/Webview.tsx

```diff
@@ -15,25 +17,27 @@ interface WebviewProps {
 export const Webview: React.FC<WebviewProps> = (props) => {
   const ref = useRef<HTMLIFrameElement>(null);
   const [spinning, setSpinning] = useState(true);
-
+  const status = useDragstatus()
```

Comment:
An important problem is component `Webview` is a pure component and should not dependency with redux. redux has a scope in main route, the other route not use redux anywhere and not need it.

This code will restrict this component must be render under redux context.


My suggestion is change pin logic, and overlay a translucent div on both sides of the drag bar. To make sure drag event not send  into iframe